mobile/library/java/org/chromium/net/impl/NativeCronvoyEngineBuilderImpl.java

```diff
@@ -227,6 +232,54 @@ public NativeCronvoyEngineBuilderImpl setUpstreamTlsSni(String sni) {
     return this;
   }
 
+  /**
+   * Set whether to use a platform specific APIs to create UDP socket and the associated QUIC packet
+   * writer. Note that `setUseV2NetworkMonitor()` also needs to be called to take effect. This is a
+   * temporary API which will be deprecated once the platform specific extension is verified to work
+   * and will be used as the default.
```

Comment:
What's the relationship between this method and setUseV2NetworkMonitor? I would have thought that using this method would enable the new writer regardless of any migration / network monitoring things.